kingpin/actors/aws/elb.py

```diff
@@ -228,3 +216,93 @@ def _execute(self):
             yield utils.tornado_sleep(3)
 
         raise gen.Return()
+
+
+class UseCert(ELBBaseActor):
+
+    """Find a server cert in IAM and use it for a specified ELB."""
+
+    all_options = {
+        'name': (str, REQUIRED, 'Name of the ELB'),
+        'region': (str, REQUIRED, 'AWS region name, like us-west-2'),
+        'cert_name': (str, REQUIRED, 'Unique IAM certificate name, or ARN'),
+    }
+
+    # Local copy of {cert_name: cert_arn} dictionary
+    cert_arn_cache = {}
+
+    def __init__(self, *args, **kwargs):
+        """Set up additional IAM connection."""
+        super(UseCert, self).__init__(*args, **kwargs)
+        self.iam_conn = boto.iam.connection.IAMConnection(
+            aws_access_key_id=aws_settings.AWS_ACCESS_KEY_ID,
+            aws_secret_access_key=aws_settings.AWS_SECRET_ACCESS_KEY)
+
+    @concurrent.run_on_executor
+    @utils.exception_logger
+    def _check_access(self, elb):
+        try:
+            # A blank ARN value should have code 'CertificateNotFound'
+            # We're only checking if credentials have sufficient access
+            elb.set_listener_SSL_certificate(443, '')
```

Comment:
The '443' here worries me. Can you make the port configurable -- because people can set their ELB to listen on 80, 443 or anything > 1024.